Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).


It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/gnucash/-/gnucash/horizontal-auto.svg)

Stephan-P committed:
Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 100.0% (5524 of 5524 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Co-authored-by: Stephan Paternotte <stephan@paternottes.net>

Commit:
Translation update  by gallegonovato <fran-carro@hotmail.es> using Weblate

po/es.po: 100.0% (5524 of 5524 strings; 0 fuzzy)
2 failing checks (0.1%)
Translation: GnuCash/Program (Spanish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/es/

Co-authored-by: gallegonovato <fran-carro@hotmail.es>

Stephan-P committed:
Translation update  by Stephan Paternotte <stephan@paternottes.net> using Weblate

po/nl.po: 100.0% (5524 of 5524 strings; 0 fuzzy)
1 failing checks (0.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Co-authored-by: Stephan Paternotte <stephan@paternottes.net>

droopy1179 committed:
Translation update  by Rene Lambers <jbh.lambers@gmail.com> using Weblate

po/nl.po: 99.9% (5523 of 5524 strings; 1 fuzzy)
1 failing checks (0.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

Co-authored-by: Rene Lambers <jbh.lambers@gmail.com>

droopy1179 committed:
Translation update  by Rene Lambers <jbh.lambers@gmail.com> using Weblate

po/nl.po: 100.0% (5524 of 5524 strings; 0 fuzzy)
1 failing checks (0.1%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

fellen commented on po/es.po:
```diff
@@ -5139,10 +5139,8 @@ msgstr "Borra el desglose efectivo"
 #. Translators: This is the label of a toolbar button. So keep it short.
 #: gnucash/gnome/gnc-plugin-page-register.c:382
-#, fuzzy
-#| msgid "Split"
 msgid "Split View"
-msgstr "Desglosar"
+msgstr "Dividir la vista"
```
@gallegonovato Thanks for informing us of an ambiguity. Sorry you have choosen the wrong meaning, Not the view is divided, but the transaction details (splits) are shown.

@CWehli Perhaps <q>{Show|Display} splits</q> would be better?
FYI [Bug 799031—Ledger types in View menu should be renamed](https://bugs.gnucash.org/show_bug.cgi?id=799031) is in the same context.

gallegonovato replied:
Good afternoon @fellen

But which chain is it? 

I say this in case you can look for another translation. But if it is an ambiguity it can cause problems, when translated?

Best regards

fellen replied:
Are you asking for [its location in Weblate](https://hosted.weblate.org/translate/gnucash/gnucash/es/?checksum=f198e8452594ea9c)?

Yes, CWehli and I  tried to resolve another ambiguity and created a new one. So i started the discussion with him.

CWehli replied:
> Perhaps "{Show|Display} splits" would be better?

`Show Split` is a good choice in my opinion. Tthe main thing is that it prevents misunderstandings.
But then we should also consider whether to adjust the German translation.

droopy1179 committed:
Translation update  by Rene Lambers <jbh.lambers@gmail.com> using Weblate

po/nl.po: 100.0% (5524 of 5524 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Dutch)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/nl/

fellen commented:
Thanks!
